templates/ai/index.html

```diff
@@ -471,6 +471,7 @@ <h2>Use modular platforms to run AI at the edge or in large clouds</h2>
         <li class="p-list__item is-ticked">Low-ops, streamlined lifecycle management</li>
         <li class="p-list__item is-ticked">A modular and open source suite for reusable deployments</li>
       </ul>
+      <a href="/blog/edge-ai" class="p-button">Read more about Edge AI</a>
```

Comment:
Shouldn't this look like a regular link instead of a button?

Reply:
Why? The other links are also buttons, it's not a lino inside of a paragraph

Reply:
@lorumic done, updated to link.